.editorconfig

```diff
@@ -0,0 +1,23 @@
+root = true
+
+[{configure,{*.{c,cpp,h,go,java,js,py,rs}}]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 4
+
+[Makefile]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = tab
+indent_size = 8
```

Comment:
Unusually wide, but otherwise not a blocker, merely an fyi

Reply:
Unusual!?

Not if you use [Linux kernel coding style](https://docs.kernel.org/process/coding-style.html#codingstyle)...

> Tabs are 8 characters, and thus indentations are also 8 characters. There are heretic movements that try to make indentations 4 (or even 2!) characters deep, and that is akin to trying to define the value of PI to be 3.

Reply:
Huh, today I learned. Thank you for the source and context :)